soa/utils/scope.h

```diff
@@ -1,4 +1,4 @@
-/* scope.h
```

Comment:
will do I had found it here: https://github.com/datacratic/soa/commit/783a13c63dd43a039757e2e69166ee909f6150e4